src/watcher.py

```
@@ -0,0 +1,81 @@
+        if self._lock.locked():
+            return
+        async with self._lock:
+            await asyncio.sleep(0.1)
+            rc = RedisCache(redis)
+            for ch in self.db.list_chats():
+                cid, style = ch['identifier'], ch['style']
+                last = self.cache.get(cid)
+                new_msgs = self.db.get_new_messages(cid, last)
+                if not new_msgs:
+                    continue
+                texts      = [m['text'] for m in new_msgs]
```

Flake8 (7.2.0)

[error] multiple spaces before operator

(E221)